lib/pbench/server/report.py

```diff
@@ -180,7 +180,7 @@ def post_status(self, timestamp, doctype, file_to_index=None):
                     fname = "report-status-payload.{}.{}.xz".format(
                         self.name, timestamp_noutc
                     )
-                    fpath = os.path.join(self.config.LOGSDIR, self.name, fname)
+                    fpath = self.config.log_dir / self.name / fname
```

Comment:
Where did `self.config.log_dir` become  a Path object? I thought it's a string.

Reply:
Yes, you are correct, the config objects `log_dir` field is always a string.  I've fixed the code to correct the inconsistent use.